src/test/java/MyTests.java

```diff
@@ -24,15 +24,17 @@ public void HelloExistTest() {
         // Open Index page
         driver.get(PATH);
         // Check hello message  shown
-        driver.findElement(By.id("header")).isDisplayed();
+        boolean isHelloMessageShown = driver.findElement(By.id("header")).isDisplayed();
+        Assert.assertTrue(isHelloMessageShown,"hello message don't show");
     }
 
     @Test
     public void BandsExistTest() {
```

Comment:
According to name convention all methods should start from lowercase letter, let's update naming